src/components/Approval/Approval.tsx

```diff
@@ -88,9 +88,10 @@ export const Approval = () => {
     moduleLogger.debug({ fees }, 'chainSpecific undefined for fees')
   }
 
-  const approvalFeeCryptoPrecision = bnOrZero(
-    fees?.chainSpecific?.approvalFeeCryptoBaseUnit ?? '0',
-  ).div(bn(10).pow(sellFeeAsset?.precision ?? 0))
+  const approvalFeeCryptoHuman = toHuman({
+    value: bnOrZero(fees?.chainSpecific?.approvalFeeCryptoBaseUnit),
+    inputPrecision: sellFeeAsset?.precision ?? 0,
```

Comment:
This is also wrong, though not in the consumption but in the definition of `toHuman()` - it returns us a precision amount, not a human amount i.e doesn't strip dp. Will get that fixed in a follow-up since there are a few consumptions we'll want to ensure are correct:

<img width="843" alt="image" src="https://user-images.githubusercontent.com/17035424/231735297-e7a91701-e68a-42c1-8ea5-524137ff3b1a.png">